### PR TITLE
Ledger's walletIndex get lost

### PR DESCRIPTION
**Pull request by veado**
- [x] Fixed / checked `Swap`, `Interactiv`, `Send`, `Deposit`, `Upgrade

Note: `walletIndex` is required on different places now (not optional as before)

Fix #1900

### CODE REVIEW

File: src/main/api/ledger/binance/transaction.ts [hunk 1]
```diff
@@ -34,7 +34,9 @@ export const send = async ({
   try {
     const clientNetwork = toClientNetwork(network)
     const prefix = getPrefix(clientNetwork)
+    console.log('walletIndex:', walletIndex)
```
Comment (gromxyz) [inline src/main/api/ledger/binance/transaction.ts]: console.log

File: src/main/api/ledger/binance/transaction.ts [hunk 2]
```diff
@@ -34,7 +34,9 @@ export const send = async ({
...
     const derivePath = getDerivePath(walletIndex)
+    console.log('derivePath:', derivePath)
```
Comment (gromxyz) [inline src/main/api/ledger/binance/transaction.ts]: console.log

File: src/renderer/components/deposit/add/SymDeposit.tsx [hunk 3]
```diff
@@ -302,7 +302,10 @@ export const SymDeposit: React.FC<Props> = (props) => {
               // Decimal needs to be converted back for using orginal decimal of this asset (provided by `assetBalance`)
               asset: convertBaseAmountDecimal(assetAmountToDepositMax1e8, assetDecimal)
             },
-            memos
+            memos,
+            // TODO (@asgx-team) Get it from props when we support Ledger7
```
Comment (gromxyz) [inline src/renderer/components/deposit/add/SymDeposit.tsx]: What's ledger 7?

Comment (veado) [reply src/renderer/components/deposit/add/SymDeposit.tsx]: Next version :sweat_smile: 
_Will remove it..._

**Review by gromxyz — APPROVED** [review]

**Pull request merged**